app/src/main/java/org/simple/clinic/home/patients/PatientsTabUiRenderer.kt

```diff
@@ -46,8 +44,8 @@ class PatientsTabUiRenderer(
         ui.showCriticalAppUpdateCard()
         ui.renderAppUpdateReason(appStalenessInMonths(model.appStaleness!!))
       }
-      model.hasLoadedNumberOfPatientsRegistered && isDrugStockReportFilledOrNull -> {
-        toggleTrainingViewVisibility(model)
+      isDrugStockReportFilledOrNull -> {
+       toggleTrainingViewVisibility(model)
       }
       model.isDrugStockReportFilled == false -> {
```

Comment:
IDE shows a warning for this condition. PTAL

Reply:
I don't see a warning? What are the actions you took to receive this?

Reply:
This is the warning I am seeing

<img width="832" alt="Screenshot 2022-11-11 at 3 12 00 PM" src="https://user-images.githubusercontent.com/6140516/201312552-f5cca7d0-652e-49f3-8bdb-578209fafad5.png">

Reply:
<img width="722" alt="Screenshot 2022-11-11 at 3 50 54 PM" src="https://user-images.githubusercontent.com/30619442/201320012-e238a325-ee86-4049-99b8-98a9bda9d7e6.png">
I don't see this in the IDE, can you clean the build once?

Reply:
FYI, we've not changed anything in the `isDrugStockReportFilled` condition.

Reply:
I don't think it's a build issue. I think the logic might have changed with recent changes. Let's sync on this and get this sorted out.